examples/acoustics_2d_radial/setrun_abl.py

```diff
@@ -0,0 +1,260 @@
+"""
+Module to set up run time parameters for Clawpack -- classic code.
+
+The values set in the function setrun are then written out to data files
+that will be read in by the Fortran code.
+
+"""
+
+from __future__ import absolute_import
+import os
+import numpy as np
+
+#------------------------------
+def setrun(claw_pkg='classic'):
+#------------------------------
+
+    """
+    Define the parameters used for running Clawpack.
+
+    INPUT:
+        claw_pkg expected to be "classic" for this setrun.
+
+    OUTPUT:
+        rundata - object of class ClawRunData
+
+    """
+
+    from clawpack.clawutil import data
+
+
+    assert claw_pkg.lower() == 'classic',  "Expected claw_pkg = 'classic'"
+
+    num_dim = 2
+    rundata = data.ClawRunData(claw_pkg, num_dim)
+
+
+    #------------------------------------------------------------------
+    # Problem-specific parameters to be written to setprob.data:
+    #------------------------------------------------------------------
+
+    probdata = rundata.new_UserData(name='probdata',fname='setprob.data')
+    probdata.add_param('rho',     1.,  'density of medium')
+    probdata.add_param('bulk',    4.,  'bulk modulus')
+
+    #------------------------------------------------------------------
+    # Standard Clawpack parameters to be written to claw.data:
+    #   (or to amrclaw.data for AMR)
+    #------------------------------------------------------------------
+
+    clawdata = rundata.clawdata  # initialized when rundata instantiated
+
+
+    # Set single grid parameters first.
+    # See below for AMR parameters.
+
+
+    # ---------------
+    # Spatial domain:
+    # ---------------
+
+    # Number of space dimensions:
+    clawdata.num_dim = num_dim
+
+    # Lower and upper edge of computational domain:
+    clawdata.lower[0] = -1.000000e+00          # xlower
+    clawdata.upper[0] = 1.000000e+00          # xupper
+    clawdata.lower[1] = -1.000000e+00          # ylower
+    clawdata.upper[1] = 1.000000e+00          # yupper
+
+    # Number of grid cells:
+    clawdata.num_cells[0] = 50      # mx
+    clawdata.num_cells[1] = 50      # my
+
+    # -------------------------
+    # Absorbing boundary layer:
+    # -------------------------
+    abldata = clawdata.abldata
+    # Select grid mapping for layer
+    #   'trigonometric', 'appelo_colonius', 'petersson_sjogreen', user defined #
+    abldata.abltype = 'trigonometric'
+    # Set depth of layer at each computation boundary
+    abldata.depth_lower[0] = 0.2
+    abldata.depth_upper[0] = 0.2
+    abldata.depth_lower[1] = 0.2
+    abldata.depth_upper[1] = 0.2
+    # Set additional parameters if needed
+    #   1) trigonometric - no additional parameters needed
+    #   2) appelo_colonius - epsilon, p, q (e.g. {'epsilon': 1e-4, 'p': 4, 'q': 4})
```

Comment:
Should perhaps add some info about these parameters to the `README.md` file, or start a new `abl.rst` file in `$CLAW/doc/doc`?